packages/checkouts/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@thirdweb-dev/checkouts",
+  "version": "1.3.3",
+  "main": "dist/thirdweb-dev-checkouts.cjs.js",
+  "module": "dist/thirdweb-dev-checkouts.esm.js",
+  "types": "dist/thirdweb-dev-checkouts.cjs.d.ts",
+  "browser": {
+    "./dist/thirdweb-dev-checkouts.esm.js": "./dist/thirdweb-dev-checkouts.browser.esm.js"
+  },
+  "exports": {
+    ".": {
+      "module": {
+        "browser": "./dist/thirdweb-dev-checkouts.browser.esm.js",
+        "default": "./dist/thirdweb-dev-checkouts.esm.js"
+      },
+      "default": "./dist/thirdweb-dev-checkouts.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "repository": "https://github.com/thirdweb-dev/js/tree/main/packages/checkouts",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/thirdweb-dev/js/issues"
+  },
+  "author": "thirdweb eng <eng@thirdweb.com>",
+  "files": [
+    "dist/"
+  ],
+  "preconstruct": {
+    "entrypoints": [
+      "index.ts"
+    ],
+    "exports": {
+      "envConditions": [
+        "browser"
+      ]
+    }
+  },
+  "sideEffects": false,
+  "dependencies": {
+    "@paperxyz/sdk-common-utilities": "*",
+    "ethers": "^5.7.2"
+  },
+  "peerDependencies": {
+    "ethers": "^5.7.2"
+  },
+  "peerDependenciesMeta": {
+    "ethers": {
+      "optional": true
+    }
+  },
+  "devDependencies": {
+    "@paperxyz/tsconfig": "*",
+    "eslint-config-paperxyz": "*",
+    "typescript": "^4.7.4"
+  },
+  "scripts": {
+    "format": "prettier --write 'src/**/*'",
+    "lint": "eslint src/",
+    "fix": "eslint src/ --fix",
+    "clean": "rm -rf dist/",
+    "build": "tsc && cross-env THIRDWEB_CHECKOUTS_SDK_VERSION=$npm_package_version preconstruct build",
```

Comment:
this env var is weird, if we need the package version we can just import the package.json wherever we need and load it that way